### ISSUE TITLE
Lack of support here is really difficult for us php devs using Shopify, we cant all switch to remix.

### ISSUE THREAD

**Issue by cthom-dev** [Shopify/shopify-app-template-php#503]
There's so many unanswered issues, going back months...just crickets.

The repo is still using Laravel 8, we are up to v11 now.

Please keep this template updated and please give us some support with issues.

**Comment by faridmovsumov** [comment]
As a PHP/Laravel dev, I tried to use this library 2-3 times but always had some problems. My production apps use this library, so I would recommend it instead. https://github.com/Kyon147/laravel-shopify

**Comment by lisen198859** [comment]
> 作为 PHP/Laravel 开发人员，我尝试过 2-3 次使用此模板，但总是遇到一些问题。我的生产应用程序使用此库，因此我推荐使用它。https [://github.com/Kyon147/laravel-shopify](https://github.com/Kyon147/laravel-shopify)

Indeed, the open-source [laravel-shopify](https://github.com/Kyon147/laravel-shopify) is more user-friendly than Shopify's official PHP SDK, especially for Laravel developers, offering a more seamless integration experience. Shopify now seems to be leaning towards recommending the Remix framework, reflecting their shift towards modern front-end technologies.

**Comment by lisen198859** [comment]
> As a PHP/Laravel dev, I tried to use this template 2-3 times but always had some problems. My production apps use this library, so I would recommend it instead. https://github.com/Kyon147/laravel-shopify

Indeed, the open-source [laravel-shopify](https://github.com/Kyon147/laravel-shopify) is more user-friendly than Shopify's official PHP SDK, especially for Laravel developers, offering a more seamless integration experience. Shopify now seems to be leaning towards recommending the Remix framework, reflecting their shift towards modern front-end technologies.
